src/const.js

```diff
@@ -0,0 +1,17 @@
+export const ActivityTypeEnum = {
+  BOOK: 0,
```

Comment:
Isn't book too specific ? This type is more about a content, like Book, Video, Article, .. ?
I propose CONTENT instead of BOOK.

Reply:
I agree, but I find that the "content" name can be confusing with the "content" field of activity ...

Reply:
What about changing the "content" field of activity to "description", and replace book by content ?